src/mpyl/project.py

```diff
@@ -270,27 +267,9 @@ class Deployment:
     def from_config(values: dict):
         props = values.get('properties')
         kubernetes = values.get('kubernetes')
-        namespace = values.get('namespace')
         traefik = values.get('traefik')
-        pr_number = os.getenv('CHANGE_ID')
-
-        if props:
-            for env_prop in props['env']:
-                for key in env_prop.keys():
-                    if '{namespace}' in env_prop[key]:
-                        if namespace is None:
-                            raise KeyError(
-                                'Found "{namespace}" placeholder but no deployment.namespace was set in project.yml'
-                            )
-
-                        env_prop[key] = re.sub('{namespace}', namespace, env_prop[key])
-
-        if traefik and pr_number:
-            for hosts in traefik['hosts']:
-                if 'pr' in hosts['host']:
-                    hosts['host']['pr'] = re.sub('{PR-NUMBER}', pr_number, hosts['host']['pr'])
```

Comment:
Could you send me and Daniel a message on Slack explaining why you solved this differently? Since we were convinced this was the best option 😬 would be a good learning opportunity.